src/cleaning/cleaning.h

```diff
@@ -41,6 +41,7 @@ struct cleaning_policy_meta {
 
 struct ocf_cleaner {
 	struct ocf_refcnt refcnt __attribute__((aligned(64)));
+	ocf_cleaning_t policy;
```

Comment:
I suggest rephrasing commit title: "store cleaning policy info in additional location outside superblock"

Reply:
How about "Additional cleaning policy info outside of the SB"? It fits the 50 characters limit.